src/test/java/RequestTest.java

```diff
@@ -0,0 +1,38 @@
+import org.junit.jupiter.api.Test;
+
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.PrintStream;
+
+
+import static http.request.RequestHandler.requestReader;
+import static http.request.RequestHandler.sendResponse;
+import static org.junit.jupiter.api.Assertions.assertArrayEquals;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class RequestTest {
```

Comment:
I see that you have two tests: 
- first one checks if the response you've hardcoded in `sendResponse` gets to mockOutput 
- the second one checks what the server received. 

I think what's missing is a complete test that checks if  X was sent to the server (in your case a GET request), Y has been sent to the client (response for the GET request)